app/controllers/api/v1/videogames_controller.rb

```diff
@@ -2,6 +2,8 @@ class Api::V1::VideogamesController < ApplicationController
   def index
     render json: Videogame.all
   end
-end
-
 
+  def show
+    render json: Videogame.find(params["id"])
```

Comment:
this is fine for now, but you will soon want to make a VideoGame serializer for this show action once you start adding reviews